plugins/blockvolume/blockprovider/gluster-block/config.go

```diff
@@ -0,0 +1,23 @@
+package glusterblock
+
+import (
+	"github.com/spf13/viper"
+)
+
+// ClientConfig holds various config information needed to create a gluster-block rest client
+type ClientConfig struct {
+	HostAddress string
+	User        string
+	Secret      string
+	CaCertFile  string
+	Insecure    bool
+}
+
+// ApplyFromConfig sets the ClientConfig options from various config sources
+func (c *ClientConfig) ApplyFromConfig(conf *viper.Viper) {
+	c.CaCertFile = conf.GetString("gluster-block-cacert")
```

Comment:
we need a configuration as listed below

* read from the parameter passed why starting glusterd2
* read from the glusterd2 toml file
* read from ENV variable